CHANGES.md

```diff
@@ -103,6 +103,7 @@ To be released.
      -  Added `StateCompleterSet<T>` struct.
      -  Added `StateCompleters<T>` static class.
      -  Added `FungibleAssetStateCompleters<T>` static class.
+ -  Added `Reorged` event on `BlockChain<T>`.  [[#945]]
```

Comment:
`ReorgedEventArgs` should be added as well.

Reply:
Rebase https://github.com/planetarium/libplanet/pull/945/commits/56c9ce1b2b3e56e32b974e28bb6a3615d72ec85e